pkg/setting/setting.go

```diff
@@ -579,8 +580,10 @@ func NewConfigContext(args *CommandLineArgs) error {
 	PluginAppsSkipVerifyTLS = Cfg.Section("plugins").Key("app_tls_skip_verify_insecure").MustBool(false)
 
 	// PhantomJS rendering
-	ImagesDir = filepath.Join(DataPath, "png")
-	PhantomDir = filepath.Join(HomePath, "tools/phantomjs")
+	phantomRendering := Cfg.Section("rendering.phantom")
+	ImagesDir = phantomRendering.Key("images_dir").MustString(filepath.Join(DataPath, "png"))
+	PhantomDir = phantomRendering.Key("phantom_dir").MustString(filepath.Join(HomePath, "tools/phantomjs"))
+	PhantomDebug = phantomRendering.Key("debug").MustBool(false)
```

Comment:
dont need this we have a logging system where each logger level can be set in config already. 

As for introducing the new settings, think we can wait with that. We will have to refactor the rendering stuff soon to switch to headless chrome and also add support for external render server. 

https://github.com/grafana/grafana/blob/master/conf/defaults.ini#L353

Reply:
@torkelo ok I can remove the extra settings, I just need to figure out if I should send `--debug=true` to phantom, how can I do that?

how close is the switch to headless chrome? after I remove the settings would it be fine to merge this anyway?

Reply:
also is there a chat where all the devs hang out?

Reply:
hm.. since you need to know the log level , might need to add a function that can return log level for a logger to this package, https://github.com/grafana/grafana/blob/master/pkg/log/log.go

Reply:
you can join our slack, https://slack.grafana.com  

We have not started on the headless chrome bit yet so not sure when we will do it :)

Reply:
@torkelo  I think you need to invite me to your slack (mikael@olenfalk.se or mikael.olenfalk@gmail.com) please :)

Reply:
you can sign up yourself, http://slack.raintank.io/   (sorry pasted wrong url before)